api/aether-top-level-openapi3.yaml

```diff
@@ -134,76 +134,265 @@ components:
     Transaction:
       description: Transaction refers to a multi-target transactional change. Taken from https://github.com/onosproject/onos-api/tree/master/proto/onos/config/v2
       properties:
+        meta:
+          description:
```

Comment:
add some simple description - maybe point to the proto file where this is defined originally e.g. `onos-api/proto/onos/config/v2/object.proto`

Reply:
done